code/src/Pages/MovieList.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+import "Css/movieList.css";
+
+const apiKey = "c616ef82836e90a0073b3a17b8bde4ba";
+
+export const MovieList = () => {
+  const [movies, setMovies] = useState([]);
+  const [category, setCategory] = useState("popular");
+
+  useEffect(() => {
+    fetch(
+      `https://api.themoviedb.org/3/movie/${category}?api_key=${apiKey}&language=en-US&page=1`
+    )
+      .then(res => res.json())
+      .then(json => {
+        console.log(json);
+        setMovies(json.results);
+      });
+  }, [category]);
+
+  return (
+    <section className="listPage">
+      <header>
+        <h1>Movie Night</h1>
+        <select value={category} onChange={e => setCategory(e.target.value)}>
+          <option value="popular">Popular</option>
+          <option value="top_rated">Top Rated</option>
+          <option value="upcoming">Upcoming</option>
+          <option value="now_playing">Now on Cinema</option>
+        </select>
+      </header>
+      <section className="movieList">
+        {movies.map(movie => (
+          <Link key={movie.id} to={`/movies/${movie.id}`}>
+            <img
+              alt="movieCard"
```

Comment:
Not a great alt text. Maybe add the actual movie name so it makes sence what movie it is?